app/views/versions/_close_ui.html.erb

```diff
@@ -18,7 +18,7 @@
     </div>
     <div class='mb-3'>
       <label for="description">Version description</label><br>
-      <textarea id="description" name="description" class='form-control'><%= @description %></textarea>
+      <textarea id="description" name="description" class="form-control" required="true"><%= @description %></textarea>
```

Comment:
I maybe should know this but I'll defer to @andrewjbtw: are there any instances where we want not to require version descriptions?

Reply:
I would reformulate this as "what should we do if no description is provided"? In the Argo interface, we should always require a description. 

I'm highly suspicious that there's an automated workflow, perhaps GIS-related based on the patterns I've seen in druids that lack version names, that does not provide a description. But that's probably out of scope here. I think if a user is prompted for a description they must provide one.

Reply:
@andrewjbtw thx!